doc/source/development/contributing_environment.rst

```diff
@@ -211,7 +211,6 @@ for :ref:`building pandas with GitPod <contributing-gitpod>`.
 Step 3: build and install pandas
 --------------------------------
 
-There are currently two supported ways of building pandas, pip/meson and setuptools(setup.py).
 Historically, pandas has only supported using setuptools to build pandas. However, this method
 requires a lot of convoluted code in setup.py and also has many issues in compiling pandas in parallel
 due to limitations in setuptools.
```

Comment:
I think we can probably remove this entire paragraph: for current readers of this document, it doesn't really matter that in the past we used setuptools, it just matters that right now we use meson

(and also the next two paragraphs could be reworded slightly)